src/main/java/com/cms/claimmanagement/insuranceCompany/repository/ClaimDetails.java

```diff
@@ -12,18 +13,103 @@
 
 @Entity
 public class ClaimDetails {
+
     @Id
     @Size(min = 10, max = 10)
-    private Long claimId;
-    //many-to-one
-    private String policyNo;
+    private String claimId;
```

Comment:
I know it doesnt change anything, but to increase readability we usually keep all of the attributes together and after the last one we create getters and setter.